entity_highlights.py

```diff
@@ -0,0 +1,73 @@
+from dbco import *
+from nltk.tokenize import sent_tokenize
+import re
+
+def create_sentences(content):
+	'''Divide content into sentences'''
+	sentences = sent_tokenize(content)
+	valid = []
+	for s in sentences:
+		
+		try:
+			# Decodes sentences names so python can work with them, removes newline characters
+			s = str(s).strip()
+			valid.append(s)
+
+		except UnicodeEncodeError:
+			pass
+
+	return valid
+
+def get_highlighted_sentences(sentences, entities):
+	'''Takes in paragraphs of text and a list of entities, returns a list of the sentences in content in which any number of words in entitites shows'''
+	words_re = re.compile("|".join(entities))
+
+	highlights = []
+
+	for sentence in sentences:
+
+		for e in entities:
+			if e.lower() in sentence.lower():
+				highlights.append(sentence)
+
+	return highlights
```

Comment:
Output not just all sentences with entities, but also which entities were in the sentence. This way, we can look up by entity later. :)

So then, the output can be `[{'sentence': 'The sentence..', entities: ['entity1', 'entity2', ...]}]`

Once we finish #9 we can make entities consist of the entity IDs.